tasks/fix-cat2.yml

```diff
@@ -3434,23 +3434,24 @@
             line: |
                 [org/gnome/settings-daemon/peripherals/smartcard]
                 removal-action='lock-screen'
-        when: rhel_08_020050_removal_action.stdout | length == 0
+        when: rhel_08_020050_removal_action.stdout_lines | length == 0
         notify: dconf update
 
       - name: "MEDIUM | RHEL-08-020050 | PATCH | RHEL 8 must be able to initiate directly a session lock for all connection types using smartcard when the smartcard is removed. | Update removal-action if exists"
         ansible.builtin.lineinfile:
             path: "{{ item }}"
             regexp: ^removal-action=
             line: removal-action='lock-screen'
-        loop: "{{ rhel_08_020050_removal_action.stdout }}"
-        when: rhel_08_020050_removal_action.stdout | length > 0
+        loop: "{{ rhel_08_020050_removal_action.stdout_lines }}"
+        when: rhel_08_020050_removal_action.stdout_lines | length > 0
         notify: dconf update
 
       - name: "MEDIUM | RHEL-08-020050 | PATCH | RHEL 8 must be able to initiate directly a session lock for all connection types using smartcard when the smartcard is removed. | Set smartcard section of db"
         ansible.builtin.lineinfile:
-            path: '/etc/dconf/db/distro.d/locks/{{ rhel_08_020050_removal_action_file.stdout }}'
+            path: '/etc/dconf/db/distro.d/locks/{{ item }}'
```

Comment:
This results in a failure message:

> “Destination /etc/dconf/db/distro.d/locks//etc/dconf/db/distro.d/20-authselect does not exist !”

The value of `item` is the full path of the file in question, as defined in `rhel_08_020050_removal_action.stdout_lines`.. So the resulting value passed to `path:` is invalid.

Using the `basename` filter on `"{{ item }}"` should fix that, I think. Testing it now, but something just like `"{{ item | basename }}"`.

Reply:
Looks like using the filter worked (the proper line was added to the target at the correct path), but I cannot push the change to this PR.

Reply:
I thought that might be a problem and was intending to test it tonight. Seems strange to me that it would just now be an issue however, since `stdout` should have contained fairly identical content previously. I'll update the PR tonight.

Reply:
Ok, I see the actual problem. Too much copy-pasting. This task needs the variable that ends in `_file` as the filenames in that variable have already had the directory stripped. I also double checked all of the changes to make sure I didn't mix any others up.